src/commands/Staff/dm.ts

```diff
@@ -45,7 +45,7 @@ const command: Command = {
 					author: { name: "Direct Message" },
 					title: `Sent message to ${recipient.tag}`,
 					description: message,
-					footer: { text: randomTip, iconURL: interaction.member.displayAvatarURL({ format: "png", dynamic: true }) },
+					footer: { text: getString("incomingDisclaimer", this.name, recipientDb.lang) },
```

Comment:
huh?

Reply:
This was probably a mistake, gonna be fixed in the latest commit